webapp/src/shared/components/Avatar/EditableAvatar.css

```diff
@@ -0,0 +1,36 @@
+.editable-avatar {
+  margin: 0;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.editable-avatar-wrapper {
+  margin: -0.5rem;
+  background-color: var(--color-light);
+  width: fit-content;
+  clip-path: inset(0.55rem 0.55rem 0.55rem 0.55rem round 0.55rem);
+  border: none;
+}
+
+.editable-avatar__image {
+  user-drag: none; /* TODO: none of these work for firefox: research */
```

Comment:
Despite the comment and that VSCode has a curly yellow line underneath these css properties, I can see no difference between Firefx and Chrome behaviors .

Reply:
There was:
as it was, I was disabling all dragging of all elements in the page, not disabling dragging only for the component that needs to have its dragging not intervened by the browser.
But it's fixed now, so should be okay.